relayer/pkg/chainlink/ocr2/client_test.go

```diff
@@ -125,6 +532,12 @@ func TestOCR2Client(t *testing.T) {
 		fmt.Printf("%+v\n", transmissions)
 	})
 
+	t.Run("get new transmission event", func(t *testing.T) {
+		events, err := client.NewTransmissionsFromEventsAt(context.Background(), contractAddress, 123)
+		require.NoError(t, err)
+		assert.Len(t, events, 15)
```

Comment:
just assert that we did in fact paginate to get 15 results. if we didn't then we'd only get 10 results